### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by AndreasAugustin** [AndreasAugustin/actions-template-sync#365] [Announcements]
## What's Changed
* docs: add pedrorrivero as a contributor for ideas by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/332
* docs: add george-gca as a contributor for ideas by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/333
* Chore(deps): Bump alpine from 3.17.2 to 3.17.3 in /src by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/335
* Chore(deps): Bump alpine from 3.17.2 to 3.17.3 by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/336
* Chore(deps): Bump node from 19.8.1-alpine to 19.9.0-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/337
* Chore(deps): Bump node from 19.9.0-alpine to 20.0.0-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/339
* Chore(deps): Bump node from 20.0.0-alpine to 20.1.0-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/340
* Chore(deps): Bump alpine from 3.17.3 to 3.18.0 by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/343
* Chore(deps): Bump alpine from 3.17.3 to 3.18.0 in /src by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/342
* docs: add ebronson68 as a contributor for ideas by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/345
* docs: add marvin-belonio as a contributor for doc by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/346
* improve github app documentation by @marvin-belonio in https://github.com/AndreasAugustin/actions-template-sync/pull/344
* Chore(deps): Bump node from 20.1.0-alpine to 20.2.0-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/347
* Chore(deps): Bump node from 20.2.0-alpine to 20.3.0-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/348
* Chore(deps): Bump alpine from 3.18.0 to 3.18.2 by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/349
* Chore(deps): Bump alpine from 3.18.0 to 3.18.2 in /src by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/350
* :sparkles: (#338) use upstream default branch instead of configured by @AndreasAugustin in https://github.com/AndreasAugustin/actions-template-sync/pull/351
* Chore(deps): Bump node from 20.3.0-alpine to 20.3.1-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/352
* :memo: (#320) add comparison with other tools by @AndreasAugustin in https://github.com/AndreasAugustin/actions-template-sync/pull/354
* doc: fixed a few typos and other small changes by @dotdc in https://github.com/AndreasAugustin/actions-template-sync/pull/355
* docs: add dotdc as a contributor for doc by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/356
* fix: checkout action using a github pat by @dotdc in https://github.com/AndreasAugustin/actions-template-sync/pull/358
* docs: add dotdc as a contributor for bug by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/360
* Chore(deps): Bump node from 20.3.1-alpine to 20.4.0-alpine by @dependabot in https://github.com/AndreasAugustin/actions-template-sync/pull/361
* docs: add dotdc as a contributor for code by @allcontributors in https://github.com/AndreasAugustin/actions-template-sync/pull/363
* feat: ability to add reviewers for pull requests by @dotdc in https://github.com/AndreasAugustin/actions-template-sync/pull/362

## New Contributors
* @marvin-belonio made their first contribution in https://github.com/AndreasAugustin/actions-template-sync/pull/344
* @dotdc made their first contribution in https://github.com/AndreasAugustin/actions-template-sync/pull/355

**Full Changelog**: https://github.com/AndreasAugustin/actions-template-sync/compare/v0.8.0...v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AndreasAugustin/actions-template-sync/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by ebronson68** [comment]
Love the new `pr_reviewers` input! Prevents the template syncing action from creating PRs that don't have any reviewers assigned to them.

**Reply by AndreasAugustin** [reply]
nice to hear :) and thx for the feedback! 🚀
